src/System.Management.Automation/engine/CommandSearcher.cs

```diff
@@ -778,19 +778,19 @@ private CommandInfo GetNextFunction()
                             _commandName,
                             WildcardOptions.IgnoreCase);
 
-                    foreach (DictionaryEntry functionEntry in _context.EngineSessionState.GetFunctionTable())
+                    foreach (var functionEntry in _context.EngineSessionState.GetFunctionTable())
```

Comment:
The type is not obvious. Please replace `var` with explicit type.

Reply:
Maybe
```csharp
ExecutionContext context = _context;
SessionStateInternal sessionState = context.EngineSessionState;
IDictionary<string, FunctionInfo> functionTable = sessionState.GetFunctionTable();
foreach (KeyValuePair<string, FunctionInfo> functionEntry in functionTable)
{
...
```
The types are not obvious in `_context.EngineSessionState.GetFunctionTable()` either...

Reply:
`_context.EngineSessionState` is common pattern in Engine. .For the small refactor I thing it makes sense only replace `var` with type.